src/pages/Home/index.jsx

```diff
@@ -1,15 +1,25 @@
 import Header from "../../components/Header";
 import MusicList from "../../components/MusicList";
+import PlayLists from "../../components/PlayLists";
 import Sidebar from "../../components/Sidebar";
 
 import "./styles.css";
 
-const Home = () => {
+const Home = ({ content, title }) => {
+  let contentElement = <MusicList />;
+  if (content === "album") {
+    contentElement = <MusicList type={content} />;
+  } else if (content === "single") {
+    contentElement = <MusicList type={content} />;
+  } else if (content === "playlists") {
+    contentElement = <PlayLists />;
+  }
```

Comment:
If you do [this](https://github.com/mim24sd/music-app/pull/8/files#r1058255975), then you can remove all of these.

Reply:
Yeees done